src/test/java/org/apache/commons/io/filefilter/FileFilterTestCase.java

```diff
@@ -1406,7 +1406,7 @@ public void testFilterListNullParameters() {
 
         final IOFileFilter filter = FileFilterUtils.trueFileFilter();
         try {
-            FileFilterUtils.filterList(filter, Arrays.asList((File) null));
+            FileFilterUtils.filterList(filter, Collections.singletonList((File) null));
```

Comment:
It is worth testing both kinds?

Reply:
@garydgregory it is only used as a Iterable there thus I think it has no actual difference IMO, exluding running speed.